server/game/AbilityTargets/AbilityTargetSelect.js

```diff
@@ -43,7 +43,7 @@ class AbilityTargetSelect {
     }
 
     getGameAction(context) {
-        let choice = this.properties.choices[context.selects[this.name].choice];
+        let choice = this.properties.choices[context.selects[this.name] && context.selects[this.name].choice];
```

Comment:
if there's no select, don't you want to return an empty array here?

Reply:
Sure.  I've adjusted as requested.